### ISSUE TITLE
SMTP and NODE_TLS_REJECT_UNAUTHORIZED

### ISSUE THREAD

**Issue by aldanor** [jhuckaby/Cronicle#42]
In order to sent notifications e-mails when SSL is self-signed, I suppose that `NODE_TLS_REJECT_UNAUTHORIZED` has to be set, otherwise it's rejected with an error. There is `web_hook_ssl_cert_bypass` option for web hooks, but no such option for e-mails, which would definitely be nice to have.

For completeness, here's what is logged in `/opt/cronicle/Error.log`:

```
[1510480800.66][2017-11-12 10:00:00][scheduler][Error][error][mail]
[Failed to send e-mail for job: jj9wl9orw0l: ...@...: Error: self signed certificate]
[...]
```

**Comment by jhuckaby** [comment]
Yup, sorry about this.  Funny enough, [web_hook_ssl_cert_bypass](https://github.com/jhuckaby/Cronicle#web_hook_ssl_cert_bypass) should actually do what you want.  That literally sets the `NODE_TLS_REJECT_UNAUTHORIZED ` flag globally at startup, so it should also affect SMTP.

I guess this is more of a config property naming and documentation bug, it would seem.  I guess that flag should be called, simply, `ssl_cert_bypass`, as it affects both web hooks and SMTP.

**Comment by jhuckaby** [comment]
Fixed in [v0.7.5](https://github.com/jhuckaby/Cronicle/releases/tag/v0.7.5).  Thanks!
